dvc/remote/local.py

```diff
@@ -152,6 +152,9 @@ def remove(self, path_info):
 
         if self.exists(path_info):
             remove(path_info.fspath)
+            # Remove unpacked if dir checksum is true
```

Comment:
I don't think that this comment is necessary, the code is pretty self-explanatory :)

Reply:
Removed the comment.